priv/repo/migrations/20170721171053_change_stage_default_value.exs

```diff
@@ -0,0 +1,9 @@
+defmodule RemoteRetro.Repo.Migrations.ChangeStageDefaultValue do
+  use Ecto.Migration
+
+  def change do
```

Comment:
hey @qlaire, can you change this migration so that there is an `up` and a `down` method? that way, we can roll this migration back to the previous default. the `change` method works in most migration cases, but here Ecto doesn't know what to roll the default back to